pkg/graphviz/traceflow.go

```diff
@@ -475,6 +484,64 @@ func GenGraph(tf *crdv1alpha1.Traceflow) (string, error) {
 		}
 		edge.Attrs[gographviz.Constraint] = "false"
 	}
-
+	if tf.Spec.LiveTraffic && tf.Status.Phase == crdv1alpha1.Succeeded {
+		err = createCapturedPacketNode(graph, cluster2.Name, tf)
+		if err != nil {
+			return "", err
+		}
+	}
 	return genOutput(graph, false), nil
 }
+
+func getCapturedPacketLabel(tf *crdv1alpha1.Traceflow) string {
+	label := "Captured Packet:\\lSource IP:" + tf.Status.CapturedPacket.SrcIP + "\\l" +
```

Comment:
Missing a space here:
```suggestion
	label := "Captured Packet:\\lSource IP: " + tf.Status.CapturedPacket.SrcIP + "\\l" +
```